List `axios` as peer dependency

Starting a new next.js project with `@storyblok/react` and running `yarn dev` results in the following error:

```
wait  - compiling...
error - ./node_modules/@storyblok/react/dist/storyblok-react.mjs:30:0
Module not found: Can't resolve 'axios'
```

I suspect this is because this package depends on `storyblok-js` which then depends on `storyblok-js-client` which requires `axios` as a peer dependency. All of these packages should probably list `axios` as a `peerDependency` in their `package.json` along with updating their `README.md` installation instructions.

--- a/README.md
+++ b/README.md
@@ -36,11 +36,11 @@
 
 ### Installation
 
-Install `@storyblok/react`:
+Install `@storyblok/react` and it's peer dependency `axios`:
 
 ```bash
-npm install @storyblok/react
-// yarn add @storyblok/react
+npm install @storyblok/react axios
+// yarn add @storyblok/react axios
 ```
 
 #### From a CDN

--- a/package.json
+++ b/package.json
@@ -27,6 +27,9 @@
     "lint-staged": "^13.0.3",
     "prettier": "^2.7.1"
   },
+  "peerDependencies": {
+    "axios": "^0.27.2"
+  },
   "lint-staged": {
     "*.{js,jsx,css,ts,tsx}": [
       "prettier --write",

